profiles/standard/standard.install

```diff
@@ -401,3 +401,20 @@ function standard_install() {
   config_set('system.core', 'preprocess_js', '1');
   config_set('system.core', 'cache', '1');
 }
+
+/**
+ * Set home page menu to render as a drop-down menu.
+ */
+function standard_update_1000() {
+  if ($layout = layout_load('home')) {
```

Comment:
Sorry I should have remembered this in the first round of feedback, but we shouldn't be using APIs like `layout_load()` in the update hook. Instead you should use config and load the file directly, e.g. `$config = config('layout.layout.home');` and then check `$config->isNew()` to see if it existed.